staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -144,6 +144,61 @@ var (
 		},
 		[]string{requestKind})
 
+	apiserverCurrentR = compbasemetrics.NewGaugeVec(
+		&compbasemetrics.GaugeOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "current_r",
```

Comment:
I think no one is going to understand these, which is fine, but maybe we can add a breadcrumb, like an "apf_" prefix. Either here or in the help line ("APF: ")?

Reply:
The `subsystem` already says "flowcontrol", which is the word we are using here instead of APF.  I will expand the help.